### ISSUE TITLE
Failure to install on gettext-setup on Ruby 2.0

### ISSUE THREAD

**Issue by elconas** [puppetlabs/gettext-setup-gem#16]
Release 0.9 of gettext-setup does not install on Centos7 (Ruby 2.0) as the dependency of fast_gettext failes to install. 

```
Gem::InstallError: fast_gettext requires Ruby version >= 2.1.0.
An error occurred while installing fast_gettext (1.3.0), and Bundler cannot continue.
Make sure that `gem install fast_gettext -v '1.3.0'` succeeds before bundling.
```

This will also cause librarian-puppet to fail. Could be caused by https://github.com/puppetlabs/gettext-setup-gem/commit/7c3b2fe7ee09136515dc84a5c9ac704e1ad2effc#diff-e79a60dc6b85309ae70a6ea8261eaf95L5

How to repoduce: 

run gem install librarian-puppet on CentOS7

**Comment by mattkenn4545** [comment]
I have this same issue.  See https://groups.google.com/forum/#!topic/puppet-users/t4p7y7Lf9qk

**Comment by smcclellan** [comment]
@elconas @mattkenn4545 This should be fixed; please try gem version 0.10.

**Comment by elconas** [comment]
Works now with 0.10
